### ISSUE TITLE
Makefile

### ISSUE THREAD

**Issue by igosat** [voodoos/Projet_Simulateur_IGOSAT#8]
Bonjour, un petit makefile pour ce qui ne travaillent pas avec des environnements de développement.

**Comment by igosat** [comment]
C'est un proposition. Je veux bien le faire pendant la semaine.

**Comment by voodoos** [comment]
Ah d'accord, j'hésitais entre requête et proposition... 
Ok donc !

Au fait, qui se cache derrière le compte igosat ?

**Comment by G-Ray** [comment]
+1
Il serait même judicieux d'utiliser cmake :)

**Comment by voodoos** [comment]
Oui, il faut pouvoir le générer automatiquement sinon on ne s'en sortira pas.

**Comment by igosat** [comment]
Muahahahaha, vous ne le saurez jamais, nan c'est Colin. cmake si je ne me trompe est un similaire de make adapter aux différentes plateformes pour résoudre les problèmes de librairies et autre c'est ça?

**Comment by igosat** [comment]
Pourquoi pas utiliser autotools plutôt que cmake, il faut installer cmake pour pouvoir utiliser compiler en utilisant make?

**Comment by voodoos** [comment]
Tu t'en sors ?
